src/theme/shared/colors.ts

```diff
@@ -64,6 +64,9 @@ const colors: Colors = {
     800: '#831E0E',
     900: '#611B0D',
   },
+  custom: {
+    btnShadow: '#C2BF00',
```

Comment:
Could you please doble check the shadow? I know that maybe in figma has this type of color which we don't have in our palette BUT we are trying to avoid adding more colors.
Sergiu has some approaches where we can make use of the color that we already have. Please doble check with Sergiu.

Thanks!